app/webpacker/components/Panel/pages/RunValidatorsPage/RunValidatorsForm.jsx

```diff
@@ -0,0 +1,158 @@
+      selectedCompetitionRange,
+      selectedValidators,
+      applyFixWhenPossible,
+    );
+  };
+
+  const {
+    data: validationOutput, isFetching, isError, refetch: runValidators,
+  } = useQuery({
+    queryKey: ['competitionCountInRange'],
+    queryFn: runValidatorsForCompetitions,
+    enabled: false,
+  }, RUN_VALIDATORS_QUERY_CLIENT);
+
+  // enableCompetitionEditor says whether competition list editor should be enabled or not. If the
+  // list of competitions is passed as parameter, then the editor need not be shown.
+  const enableCompetitionEditor = !competitionIds;
+
+  // Competition name needs to be shown on output only when the script is not ran just for a single
+  // competition.
+  const showCompetitionNameOnOutput = !(
+    selectedCompetitionSelectionOption === COMPETITION_SELECTION_OPTIONS_MAP.manual.key
+    && selectedCompetitionIds.length === 1
+  );
+
+  return (
+    <>
+      <Form onSubmit={runValidators}>
+        {enableCompetitionEditor && (
+          <>
+            <Header>Competition Selector</Header>
+            <FormGroup grouped>
+              {COMPETITION_SELECTION_OPTIONS.map((option) => (
+                <FormField key={option}>
+                  <Radio
+                    label={COMPETITION_SELECTION_OPTIONS_MAP[option].text}
+                    name="competitionSelectionOption"
+                    value={option}
+                    checked={selectedCompetitionSelectionOption === option}
+                    onChange={setSelectedCompetitionSelectionOption}
+                  />
+                </FormField>
+              ))}
+            </FormGroup>
+            {(
+              selectedCompetitionSelectionOption === COMPETITION_SELECTION_OPTIONS_MAP.manual.key
+            ) && (
+            <Form.Field
+              label="Competition ID(s)"
+              control={IdWcaSearch}
+              name="competitionIds"
+              value={selectedCompetitionIds}
+              onChange={setSelectedCompetitionIds}
+              model={SEARCH_MODELS.competition}
+              multiple
+              required
+            />
+            )}
+            {(
+              selectedCompetitionSelectionOption === COMPETITION_SELECTION_OPTIONS_MAP.range.key
+            ) && (
+              <CompetitionRangeSelector
+                range={selectedCompetitionRange}
+                setRange={setSelectedCompetitionRange}
+              />
+            )}
+          </>
+        )}
+        <Header>Run Validators</Header>
+        <Form.Dropdown
+          label="Select Validators to run"
+          options={VALIDATOR_OPTIONS}
+          fluid
+          multiple
+          selection
+          value={selectedValidators}
+          onChange={setSelectedValidators}
+        />
+        <Form.Checkbox
+          label={`Apply fix when possible (List of validators with automated fix: ${
+            VALIDATORS_WITH_FIX.map(validatorNameReadable).join(', ')
+          })`}
```

Comment:
Bit awkward to have such a (potentially) long list as part of the checkbox label. Can you add them as an extra text paragraph below?

Reply:
SemUI `Header` as `h4` or `h5` with only the `subheader` and no "real" header should do: https://react.semantic-ui.com/elements/header/#content-subheader

Reply:
Done